flink-core/src/main/java/org/apache/flink/configuration/TaskManagerOptions.java

```diff
@@ -708,6 +708,11 @@ public class TaskManagerOptions {
                                                     "The %s mode tries to spread out the slots evenly across all available %s.",
                                                     code(TaskManagerLoadBalanceMode.SLOTS.name()),
                                                     code("TaskManagers")),
+                                            text(
```

Comment:
You have to regenerate the documentation now if I'm not mistaken (docs/README.md). 🤔

Reply:
I was sure I was forgetting sth :D should be addressed now